smarts/sstudio/genscenario.py

```diff
@@ -434,7 +434,8 @@ def gen_traffic_histories(scenario: str, histories_datasets, overwrite: bool):
     if os.path.exists(road_network_path):
         from smarts.core.sumo_road_network import SumoRoadNetwork
 
-        road_network = SumoRoadNetwork.from_file(road_network_path)
+        map_spec = types.MapSpec(road_network_path)
+        road_network = SumoRoadNetwork.from_spec(map_spec)
```

Comment:
Why is there a need to re-create `sumo_road_network`

Reply:
Good point.  This is tricky because `gen_traffic_histories()` can be called independently or from `gen_scenario()`, and it might have been created in `gen_scenario()`.   I'll make it a parameter with a default value to solve this.

Reply:
Done.